source/Cucumber.SimpleDb/Transport/AwsRestService.cs

```diff
@@ -17,6 +17,7 @@ internal class AwsRestService : IAwsRestService
         private readonly string _publicKey;
         private readonly string _privateKey;
         private readonly IWebRequestProvider _webRequest;
+        private readonly string _simpleDbUrl = "sdb.amazonaws.com";
         private static readonly XNamespace sdbNs = "http://sdb.amazonaws.com/doc/2009-04-15/";
 
         public AwsRestService(string publicKey, string privateKey, IWebRequestProvider webRequest)
```

Comment:
This ctor overload should be modified to call `: this(publicKey, privateKey, "sdb.amazonaws.com", webRequest)` and the body removed, so there is only one ctor execution path; the private member field should not have a default value.